icons/h&m.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="101.333" height="66.667" version="1.0" viewBox="0 0 76 50"><path d="M17.7 3.2c-.8 1.3-3 6.2-4.8 11l-3.4 8.7-4.9 1.2-5 1.2 3.3 2.9 3.2 2.9-2.2 6.7c-1.2 3.7-2.4 7.3-2.7 8-.2.6.2 1.2 1 1.2s2.6-3.2 4.3-8.2c3-8.1 3.1-8.2 7.4-9.5 2.5-.7 4.6-1.1 4.8-1 .2.2-.8 3.7-2.2 7.7-3 8.8-3.1 10.2-.6 12.4 1.9 1.7 2 1.7 3.4-1.5.7-1.9 2.6-7.1 4.2-11.7 1.8-5.3 3.6-8.7 4.9-9.4 2.2-1.2 5.5-4.8 5.6-6.1 0-.4-.9-.7-2-.7s-2-.2-2-.4 1.4-4.1 3.1-8.6c1.7-4.5 2.9-8.4 2.6-8.7-.3-.3-1.8-.1-3.4.4-2.3.8-3.4 2.4-6.3 9.6l-3.5 8.6-4.5 1c-2.5.5-4.6.8-4.8.7-.1-.2.7-2.5 1.9-5.2C19.7 5.7 20.9 2.1 20 1.5c-.5-.3-1.6.5-2.3 1.7zM52.3 2.5c-3 1.3-3.5 2.3-9.1 19.7-7.1 21.7-7.9 25.2-6.1 24.6.7-.2 3.5-7.3 6.2-15.7 2.6-8.4 5-15.1 5.1-15 .2.2-.1 5.5-.7 11.7-.9 10.7-.9 11.5 1 14 1.1 1.5 2.4 2.6 2.9 2.4.5-.1 3.4-6.2 6.4-13.5 3.1-7.3 6.1-14.1 6.8-15.2l1.3-2-.5 2c-.3 1.1-1.7 8.3-3.1 16L60 45.4l2.4 2.5c3.1 3 2.4 4.9 7.7-20.9 5.3-26 5.3-26 4.1-26-5.3 0-7.4 3.1-15.8 22.5-5 11.5-5.4 10.4-2.9-7C57.1 5.2 57.3 1 56.3 1c-.5.1-2.2.7-4 1.5zM29.7 29.6c-.4.4-.7 1.6-.7 2.6s-.7 3-1.6 4.3c-2.3 3.5-1.2 5.5 3 5.3 2.3-.1 3.3-.5 2.9-1.4-.2-.6 0-1.5.6-1.9 1.7-1 1.3-2.8-.5-2.1-1.3.5-1.5.2-1-1.6 1.2-4 1.1-5.2-.4-5.5-.9-.2-1.9-.1-2.3.3zM31 38c0 .5-.4 1-1 1-.5 0-1-.5-1-1 0-.6.5-1 1-1 .6 0 1 .4 1 1z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>HandM</title><path d="M16.866 4.565c.871-.43 1.303-.372 1.313.061.012.565-.071 1.313-.131 1.857-.323 2.947-.864 5.356-.906 8.383 1.424-3.686 2.619-6.248 4.156-9.403.488-1.004.8-.818 1.2-.996 1.563-.695 1.622-.268 1.417.58-.758 3.151-2.695 13.069-2.992 14.596-.086.442-.566.255-.69.082-.557-.77-1.184-.781-1.113-1.293.348-2.546 1.603-8.934 1.93-10.439-1.669 3.42-3.398 7.694-4.286 10.118-.189.515-.531.477-.745.1-.3-.529-.883-.799-.98-1.424-.305-1.987.349-5.776.44-8.172-.908 2.618-2.433 7.7-3.1 9.997-.277.95-1.194.795-.95-.13C12.444 14.626 14.63 7.81 15.577 5.36c.222-.576.805-.556 1.289-.795zm-5.535-.474c.276.067.254.393.085.838-.499 1.314-1.19 3.04-1.98 5.076.496-.057.766-.076.766-.076.713-.088.846.269.645.648-.16.303-.353.101-1.02 1.096-.376.561-.99.71-1.332.784a183.23 183.23 0 0 0-2.557 7.188.339.339 0 0 1-.598.142c-.402-.562-.698-.58-1.01-1.034-.037-.074-.11-.173-.066-.354.229-.936.837-2.86 1.762-5.392-1.165.253-2.426.524-2.947.645a234.37 234.37 0 0 0-1.81 4.944c-.359 1.015-1.29.819-.964-.14A167.39 167.39 0 0 1 1.978 13.8c-.646-.069-.864-.525-1.225-.906-.14-.147-.443-.126-.592-.332-.267-.371-.24-.536.377-.714A61.993 61.993 0 0 1 2.97 11.2c1.044-2.694 1.987-5.006 2.49-6.24.388-.95 1.387-.861 1.008.076a461.123 461.123 0 0 0-2.344 5.893 57.172 57.172 0 0 1 2.904-.576c.713-1.83 1.462-3.648 2.247-5.45.055-.125.215-.303.381-.331.525-.09 1.38-.552 1.675-.48zM9.543 16.426c.082-.05.163-.11.244-.178a49.062 49.062 0 0 1-.183-.55c-.06.078-.12.157-.179.237-.35.48-.142.65.118.491zm.465-2.275c.433-.489-.423-.693-.226.054a7.9 7.9 0 0 0 .04.146 7.09 7.09 0 0 0 .186-.2zm.45 1.473c.429-.372.838-.003.414.624-.09.135-.194.28-.307.424.045.128.091.255.14.381.219.562-.357.703-.542.237l-.05-.13c-.35.31-.754.506-1.186.371-.711-.22-.892-1.222-.228-1.993.266-.31.466-.524.634-.695-.041-.133-.08-.267-.118-.4-.087-.313-.164-.676.148-1.047.586-.695 1.945-.078 1.257.992a8.225 8.225 0 0 1-.56.748l.21.66a5.08 5.08 0 0 1 .188-.172z"/></svg>
```

This is completely wrong. 
You can merge my suggestion or redo. Please review contributing guidelines.